index.php

```diff
@@ -100,7 +100,7 @@
 	}
 }
 
-if ((x($_SESSION, 'language')) && ($_SESSION['language'] !== $lang)) {
+if (x($_SESSION, 'language') && ($_SESSION['language'] !== $lang)) {
```

Comment:
Can you please rewrite `x()` to `!empty()`?

Reply:
I'm just removing notices in this task. I would like to delegate the other task to you, if you don't mind ;-)